source/ceylon/test/engine/internal/file.ceylon

```diff
@@ -0,0 +1,39 @@
+import java.io {
+    File,
+    FileOutputStream,
+    PrintWriter
+}
+
+native class FileFacade(String path) satisfies Destroyable {
```

Comment:
There is already file facade https://github.com/ceylon/ceylon-sdk/blob/master/source/ceylon/test/reporter/HtmlReporter.ceylon#L247